pyroscope_backends/pyroscope_pyspy/src/lib.rs

```diff
@@ -29,7 +30,7 @@ pub fn pyspy_backend(config: PyspyConfig) -> BackendImpl<BackendUninitialized> {
 #[derive(Debug, Clone)]
 pub struct PyspyConfig {
     /// Process to monitor
-    pid: Option<i32>,
+    pid: Option<Pid>,
```

Comment:
Use the type `remoteprocess::Pid` that is dynamic depending on the target platform.